packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts

```diff
@@ -41,8 +41,8 @@ const source = "bicep-envchecker";
 export async function ensureBicep(ctx: SolutionContext): Promise<string> {
   const bicepChecker = new BicepChecker(ctx.logProvider, ctx.telemetryReporter);
   try {
-    if (bicepChecker.isEnabled() && !bicepChecker.isInstalled()) {
-      bicepChecker.install();
+    if ((await bicepChecker.isEnabled()) && !(await bicepChecker.isInstalled())) {
```

Comment:
I checked and found that the BicepEnvCheckerEnable feature flag default value is false. Should we turn it as true by default?

Reply:
It's not ready to release and need more testing.